src/EditorFeatures/Core/Implementation/Outlining/AbstractOutliningTaggerProvider.RegionTag.cs

```diff
@@ -23,14 +23,15 @@ protected class RegionTag : IOutliningRegionTag
             private const int MaxPreviewText = 1000;
 
             private readonly AbstractOutliningTaggerProvider<TRegionTag> _provider;
-            private readonly OutliningSpan _outliningSpan;
 
             private readonly ITextBuffer _subjectBuffer;
             private readonly ITrackingSpan _hintSpan;
 
-            public bool IsDefaultCollapsed => _outliningSpan.IsDefaultCollapsed;
-            public bool IsImplementation => _outliningSpan.AutoCollapse;
-            public object CollapsedForm => _outliningSpan.BannerText;
+            public bool IsDefaultCollapsed => OutliningSpan.IsDefaultCollapsed;
+            public bool IsImplementation => OutliningSpan.AutoCollapse;
+            public object CollapsedForm => OutliningSpan.BannerText;
+
+            protected readonly OutliningSpan OutliningSpan;
```

Comment:
i have no problem changing to that.  But, out of curiosity, why?  We've done protected fields quite a bit.  For readonly data, i don't mind them at all.  Thoughts?